projector_IP: "192.168.10.10"
server_IP: "192.168.10.11"
connection_port: 9090
license_file: "my_license_file.lic"